src/rotational_force.cpp

```
@@ -3,44 +3,312 @@
+      }
+    }
   }
+}
+
+void visualize_rotational_force(
+    const std::vector<std::vector<Eigen::Vector2f>>& rotational_force_storage,
+    const nav_msgs::msg::OccupancyGrid& gridmap,
+    const rclcpp::Publisher<visualization_msgs::msg::MarkerArray>::SharedPtr& marker_pub,
+    rclcpp::Logger logger,
+    rclcpp::Clock::SharedPtr clock,
+    std::string frame_id)
```

cppcheck

[performance] Function parameter 'frame_id' should be passed by const reference.

(passedByValue)